src/main/java/de/heuremo/commons/common/helpers/SharedPreferenceHelper.java

```diff
@@ -0,0 +1,85 @@
+package de.heuremo.commons.common.helpers;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import de.heuremo.commons.common.constant.SharedPreferencesConstant;
+
+public
+class SharedPreferenceHelper {
+
+
+    public static Boolean saveCustomerNumberAndKey(Context context, String customerNumber, String invitationKey) {

```

Comment:
method should save one value in shared preferences, make another for remaing value